src/tests/EdFi.DataManagementService.Tests.E2E/Features/RESTAPIConventions/RESTAPIConventions.feature.disabled

```diff
@@ -0,0 +1,84 @@
+Feature: No Authenticated - No Authorized REST API Conventions
```

Comment:
Is this title right? Because the user is authenticated and authorized

Reply:
Changed